subprojects/platform-jvm/src/test/groovy/org/gradle/jvm/toolchain/install/internal/JdkCacheDirectoryTest.groovy

```diff
@@ -47,10 +50,17 @@ class JdkCacheDirectoryTest extends Specification {
 
     def "lists jdk directories when listing java homes"() {
         given:
-        def jdkCacheDirectory = new JdkCacheDirectory(newHomeDirProvider(), Mock(FileOperations))
+        def jdkCacheDirectory = new JdkCacheDirectory(newHomeDirProvider(), Mock(FileOperations), mockLockManager())
         def install1 = temporaryFolder.newFolder("jdks/jdk-123")
+        new File(install1, "provisioned.ok").createNewFile()
```

Comment:
I would use the constant here, especially as it is `public`

Reply:
Oh, actually that constant was supposed to be private. Generally, I try to avoid having such dependencies (constants, ..) between prod and test code. Test code should exercise a class by its behavior, not the implementation. Shall I change the constant in my prod code, my test should actually fail because I changed the behavior (it now requires a different file to be present). Anyway, just my take on it. Can fix it either way.